kart/export.py

```diff
@@ -0,0 +1,41 @@
+import click
+
+from kart.completion_shared import repo_path_completer
+from kart.cli_util import KartCommand, forward_context_to_command
+
+
+# Aliases `kart export` to `kart table-export`.
+# One day we might support other types of export, and when we do, this will then
+# be a general purpose command that delegates to one or other more specific commands.
+# See: how `kart import` delegates to one of table-import, point-cloud-import, raster-import.
```

Comment:
Some formats can do vectors and rasters (eg: GeoPackage, FileGDB)... how can/should we deal with that in future?